File: Modul-1/soal-4/PRAK104-2410817210017-OrlandoSugian.c
#include <stdio.h>

int main()
{ 
    float A, B, c, d;
    A = 400000;
    B = 350000;
    c = 0.13;
    d = 0.21;
    printf("Harga sepatu A adalah %.0f\n", A);
    printf("Harga sepatu B adalah %.0f\n", B);
    float diskonA = A - ( A * c );
    float diskonB = B - ( B * d );
    printf("Sepatu A mendapat diskon 13%% sehingga hasilnya menjadi %.0f\n", diskonA);
    printf("Sepatu B mendapat diskon 21%% sehingga hasilnya menjadi %.0f", diskonB);
    return 0;
}

File: Modul-1/soal-4/PRAK104-2410817210017-OrlandoSugian.py
A = 400000
B = 350000
c = 0.13
d = 0.21
print(f"Harga sepatu A adalah {A}")
print(f"Harga sepatu B adalah {B}")

harga_setelah_diskon_A = A - (A * c)
harga_setelah_diskon_B = B - (B * d)

print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {harga_setelah_diskon_A :.0f}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {harga_setelah_diskon_B :.0f}")
